jobs/telemetry_batch_view.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+from os import chdir
+from os import environ
+from subprocess import call
+from sys import stderr
+import json
+
+if not environ.get('class'):
```

Comment:
Errors in the configuration should be found at dag definition time. The CI should catch those and prevent us from deploying broken jobs.

Reply:
fixed